packages/app/web/index.html

```diff
@@ -45,6 +45,7 @@
 
       body {
         display: flex;
+        flex-direction: column;
```

Comment:
Any preference on having the banner block level vs. `position: absolute` instead?